rc4airconbotmvp.py

```diff
@@ -220,9 +202,7 @@ def prompt_end_buttons(update, context):
         text=text
     )
 
-    # can input their current room information and lower credit limit
```

Comment:
For the line `Your lower credit limit has been set to $" + str(Decimal(user_input)) + ". \nInformation has been updated`, use `%.2f` to format the currency